packages/components/form/src/form-item.vue

```diff
@@ -258,16 +256,11 @@ export default defineComponent({
         path = path.replace(/:/, '.')
       }
       const prop = getPropByPath(model, path, true)
-      validateDisabled.value = true
       if (Array.isArray(value)) {
         prop.o[prop.k] = [].concat(initialValue)
       } else {
         prop.o[prop.k] = initialValue
       }
-      // reset validateDisabled after onFieldChange triggered
-      nextTick(() => {
-        validateDisabled.value = false
-      })
     }
 
     const getRules = () => {
```

Comment:
I was just wondering can we make this one as `computed` since we could be invoking this for a lot of times.

NB: we can and we should put this into separate PRs

Reply:
Yes, i agree, sth we can do in another PR :)